app/models/course/discussion/post.rb

```diff
@@ -5,7 +5,7 @@ class Course::Discussion::Post < ApplicationRecord
 
   acts_as_forest order: :created_at, optional: true
   acts_as_readable on: :updated_at
-  has_many_attachments
+  has_many_attachments on: :text
```

Comment:
This was added in https://github.com/Coursemology/coursemology2/pull/941, but doesn't seem to have been used. I'm trying to verify that discussion posts don't actually have any attachments right now.

Reply:
A few hundred discussion posts do have attachments, they should be forum posts. Will these be affected?

Reply:
I will do a quick check later to check this out!

Reply:
Need to do a DB migration to migrate some of the attachment references that were created based on the design in #941.

Reply:
As discussed, that might not be sufficient as the S3 URLs have an expiry (https://docs.aws.amazon.com/AmazonS3/latest/API/sigv4-query-string-auth.html).

This would cause image attachments in emails to behave like snapchat, with the longest expiry at 7 days.